lib/3scale_toolbox/commands/import_command/openapi.rb

```diff
@@ -27,16 +27,14 @@ def self.command
 
               option  :d, :destination, '3scale target instance. Format: "http[s]://<authentication>@3scale_domain"', argument: :required
               option  :t, 'target_system_name', 'Target system name', argument: :required
+              flag    nil, 'activedocs-hidden', 'Create ActiveDocs in hidden state'
```

Comment:
Isn't it weird that just above it is with underscores and this is with a dash?

Reply:
```

Reply:
GNU style guide says dashes.
```
GNU adds long options to these conventions. Long options consist of ‘--’ followed by a name made of alphanumeric characters and dashes. Option names are typically one to three words long, with hyphens to separate words. Users can abbreviate the option names as long as the abbreviations are unique.
```

`target_system_name` is a mistake, but cannot be changed now

Reply:
I see there are already two other instances of using dashes, so it is already a mixed bag. Ok.